assets/src/edit-story/theme.js

```diff
@@ -47,6 +47,8 @@ const theme = {
     mg: {
       v1: '#616877',
       v2: '#DADADA',
+      v3: '#3C4043',
+      v4: '#80868B',
```

Comment:
Similarly this color is only ever used in combination with an opacity of .55 resulting in `#b9bcbf`, which is almost identical to `fg.v4` at `#b3b3b3`. This variable here can be removed.

Reply:
ditto here, let's still use the exact color but try to get it from the existing colors and remove these two added colors

Reply:
Use the exact color and opacity from figma and used rgba from polished.